F-FrontEnd/src/C-expr.h

```diff
@@ -19,13 +19,8 @@
 typedef int64_t     omllint_t;
 typedef long double omldouble_t;
 
-#if ADDR_IS_64 == 0
-    #define OMLL_DFMT "%lld"
-    #define OMLL_XFMT "%llx"
-#else
-    #define OMLL_DFMT "%ld"
-    #define OMLL_XFMT "%lx"
-#endif
+#define OMLL_DFMT "%lld"
```

Comment:
@clementval Does this old code cause any warnings? It looks to me that it works well as long as ADDR_IS_64 is set properly.

Reply:
@h-murai It was causing warnings on my side with gcc-7. Let me try again...

Reply:
@h-murai It gives me warning on my machine... maybe ADDR_IS_64 is not well set. But by the way, `omllint_t` is `int64_t` in any case so it makes sense to set also `OMLL_DFMT`... no?

Reply:
@clementval On my environment, ADDR_IS_64 is set to 1 and  OMLL_DFMT to `"%ld"`.

Actually, the code below causes the warning, so OMLL_DFMT should be `"%ld"`.

```c
int64_t a;
printf("%lld\n", a);
```

Reply:
Ok, will check why on my env is opposite